mne/io/base.py

```diff
@@ -2570,6 +2570,27 @@ def concatenate_raws(raws, preload=None, events_list=None, *,
         return raws[0], events
 
 
+def match_channel_orders(raws, copy=True):
+    """Ensure consistent channel order across raws.
+
+    Parameters
+    ----------
+    raws : list
+        List of `~mne.io.Raw` instances to order.
+    %(copy_df)s
```

Comment:
this won't work unless the function is decorated with `@fill_doc` or `@verbose` (here use `@fill_doc` because there's no `verbose` parameter)